### ISSUE TITLE
Unit tests

### ISSUE THREAD

**Issue by galbwe** [codeforkansascity/meep-backend#10]
Create a test directory and start adding tests for the endpoints that are already in the app. See [pytest](https://docs.pytest.org/en/latest/) documentation to get started. Also, [here's](https://www.patricksoftwareblog.com/testing-a-flask-application-using-pytest/) a good blog post on testing flask apps with pytest.

**Comment by evanharmon1** [comment]
I'll see if I can get some pytest tests going next. Paul said he will add me on GitHub but not sure when.

**Comment by evanharmon1** [comment]
I got a basic pytest setup going. I'll start working on some endpoint tests. Let me know if I'm on the wrong track with something. My commit: https://github.com/evanharmon1/meep-backend/commit/cf5c117e3b706169e18312a12873f17113d1c403

**Comment by galbwe** [comment]
Cool. Thanks @evanharmon1. I will take a look and give some feedback.

**Comment by galbwe** [comment]
@evanharmon1 Thanks for your PR. I think the next orders of business here should be
  1. restructuring the app code into its own src folder,
  2. writing a fixture that populates the test database upon startup and empties the database after the 
      tests have run,
  3. creating a client fixture so endpoints can be called,
  4. creating tests for each endpoint for one of the apis to start, maybe projects

**Comment by evanharmon1** [comment]
Makes sense. I can take a stab at restructuring the code with a src folder and the tree structure you mentioned earlier. I'll have some time tomorrow and I'll hopefully have some progress to contribute.
